engine/src/main/resources/assets/shaders/post_vert.glsl

```diff
@@ -16,7 +16,7 @@
 
 void main()
 {
-	gl_Position = ftransform();
+	gl_Position = gl_Vertex;
+//    gl_FrontColor = gl_Color;
```

Comment:
Commented code?
What about Color now?

Reply:
It's not used, and it doesn't make sense why you need to do this  for a screen buffer affect